src/components/Compose/ScheduleTaskList.tsx

```diff
@@ -0,0 +1,194 @@
+                        <span className="ml-1">
+                            <Link href={profileUrl}>@{post?.author.handle}</Link>
+                        </span>
+                    </Trans>
+                );
+            case 'quote':
+                return <Trans>QUOTE</Trans>;
+        }
+    }, [displayInfo]);
+
+    const post = first(displayInfo.posts);
+    const content = post ? readChars(post.chars, 'visible') : '';
+
+    const isFailed = task.status === 'fail';
+
+    const handleRemove = useCallback(async () => {
+        try {
+            if (!task.uuid) return;
+            const confirmed = await ConfirmModalRef.openAndWaitForClose({
+                title: t`Delete`,
+                content: (
+                    <div className="text-fourMain">
+                        <Trans>This can’t be undone and the scheduled send will be canceled.</Trans>
+                    </div>
+                ),
+                confirmButtonText: t`Confirm`,
+            });
+
+            if (!confirmed) return;
+            const result = await FireflySocialMediaProvider.deleteSchedulePost(task.uuid);
+            if (!result) return;
+            queryClient.refetchQueries({
+                queryKey: ['schedule-tasks', fireflySessionHolder.session?.profileId],
+            });
+        } catch (error) {
+            if (error instanceof Error) {
+                enqueueErrorMessage(error.message);
+            }
+        }
+    }, [task.uuid]);
+
+    return (
+        <div className="border-b border-line py-3">
+            <div className="flex items-center justify-between">
+                <div
+                    className={classNames('text-[12px] font-bold', {
+                        'text-danger': isFailed,
+                        'text-secondary': !isFailed,
+                    })}
+                >
+                    {title}
+                </div>
+                <Trash className="h-5 w-5 cursor-pointer text-secondary" onClick={handleRemove} />
+            </div>
+            <div
+                className="my-2 cursor-pointer text-fourMain"
+                onClick={async () => {
+                    const time = await ScheduleModalRef.openAndWaitForClose({
+                        type: 'update',
+                        disableClear: true,
+                    });
+                    try {
+                        if (time && time !== 'clear') {
+                            const result = await FireflySocialMediaProvider.updateSchedulePost(task.uuid, time);
+                            if (!result) return;
+                            queryClient.refetchQueries({
+                                queryKey: ['schedule-tasks', fireflySessionHolder.session?.profileId],
+                            });
+                        }
+                    } catch (error) {
+                        if (error instanceof Error) {
+                            enqueueErrorMessage(error.message);
+                        }
```

Comment:
Throw the original error.

```suggestion
                        }
                        throw error;
```